sysdig/resource_sysdig_secure_policy.go

```diff
@@ -225,12 +231,23 @@ func addActionsToPolicy(d *schema.ResourceData, policy *secure.Policy) {
 	if captureAction := d.Get("actions.0.capture").([]interface{}); len(captureAction) > 0 {
 		afterEventNs := d.Get("actions.0.capture.0.seconds_after_event").(int) * 1000000000
 		beforeEventNs := d.Get("actions.0.capture.0.seconds_before_event").(int) * 1000000000
-		policy.Actions = append(policy.Actions, secure.Action{
-			Type:                 "POLICY_ACTION_CAPTURE",
-			IsLimitedToContainer: false,
-			AfterEventNs:         afterEventNs,
-			BeforeEventNs:        beforeEventNs,
-		})
+		name := d.Get("actions.0.capture.0.name").(string)
+		if len(name) <= 0 {
+			policy.Actions = append(policy.Actions, secure.Action{
+				Type:                 "POLICY_ACTION_CAPTURE",
+				IsLimitedToContainer: false,
+				AfterEventNs:         afterEventNs,
+				BeforeEventNs:        beforeEventNs,
+			})
+		} else {
+			policy.Actions = append(policy.Actions, secure.Action{
+				Type:                 "POLICY_ACTION_CAPTURE",
+				IsLimitedToContainer: false,
+				AfterEventNs:         afterEventNs,
+				BeforeEventNs:        beforeEventNs,
+				Name:                 name,
+			})
+		}
```

Comment:
Both the if and the else are similar in case the Name is `""`, you can get rid of the if/else and just use the append.
```suggestion
		policy.Actions = append(policy.Actions, secure.Action{
			Type:                 "POLICY_ACTION_CAPTURE",
			IsLimitedToContainer: false,
			AfterEventNs:         afterEventNs,
			BeforeEventNs:        beforeEventNs,
			Name:                 name,
		})
```

Reply:
just committed your proposed change which addressed this and the previous comment